src/client/coll/mod.rs

```diff
@@ -101,24 +160,83 @@ impl<'a> Collection<'a> {
         Ok(cursor.next())
     }
 
+    // Helper method for all findAndModify commands.
+    fn find_and_modify(&self, cmd: &mut bson::Document,
```

Comment:
From [findAndModify docs](http://docs.mongodb.org/manual/reference/command/findAndModify/):
> You cannot specify a write concern to findAndModify to override the default write concern whereas, starting in MongoDB 2.6, you can specify a write concern to the update() method.

Not sure why the command doesn't allow a specified WriteConcern.